lib/services/users.pm

```diff
@@ -165,7 +168,12 @@ sub full_users_check {
     }
     else {
         #swtich to new added user then switch back
-        switch_users;
+        if (check_var('ARCH', 's390x')) {
```

Comment:
more comments need why we handle s390 different

Reply:
See PR description, comments#2 of https://bugzilla.suse.com/show_bug.cgi?id=1185943

Reply:
I mean put comment into the code, not the PR

Reply:
Update code with the comments for the reason.